Extensions/quick_tags.js

```diff
@@ -84,12 +84,12 @@ XKit.extensions.quick_tags = new Object({
 
 		XKit.interface.post_window_listener.add("quick_tags", XKit.extensions.quick_tags.post_window);
 
-		// don't run this if we're on the new post 'page'
+		// don't run this if we're on the new post iframe
 		if ($("[data-id]").length && XKit.page.react) {
 			XKit.interface.react.create_control_button("xkit-quick-tags", this.button_icon, "Quick Tags!", "", this.button_ok_icon);
-			XKit.post_listener.add("quick_tags", XKit.extensions.quick_tags.do_posts);
+			XKit.post_listener.add("quick_tags", async function() { await XKit.extensions.quick_tags.do_posts() });
 
-			this.do_posts();
+			await this.do_posts();
```

Comment:
`await` isn't necessary if the function doesn't return a value; you can pass it in to the post listener and call it as normal